alphapept/gui/start.py

```diff
@@ -1,5 +1,6 @@
 import streamlit as st
 from alphapept.gui.utils import markdown_link
+from alphapept.display import get_latest_version
```

Comment:
We have a similar function already in check_github_version() in alphapept/utils, so maybe we can recycle this function.

Reply:
Ahhh! I'd looked in alphapept/gui/utils but not alphapept/utils! Got it - I'll use that :)